### ISSUE TITLE
HAR file

### ISSUE THREAD

**Issue by kildarr** [Huachao/vscode-restclient#561]
Is the har file generated?  If so where is it?

If it is not generated can it be?

Example:

{
  "log" : {
    "version" : "1.2",
    "pages" : [ ],
    "entries" : [ {
      "startedDateTime" : "2020-04-17T15:28:27.331Z",
      "request" : {
        "method" : "GET",
        "url" : "https://XXXXXXX-configurations.cloudhub.io/v1/clients/8c9c26a9-3648-48ac-88b9-5545fce2261b/environments",
        "httpVersion" : "",
        "cookies" : [ ],
        "headers" : [ {
          "name" : "Authorization",
          "value" : "Bearer 964da377-3d2c-429f-b158-abc9eb3c49d1"
        } ],
        "queryString" : [ ],
        "headersSize" : 0,
        "bodySize" : 0,
        "comment" : ""
      },
      "response" : {
        "status" : 200,
        "statusText" : "",
        "httpVersion" : "",
        "cookies" : [ ],
        "headers" : [ {
          "name" : null,
          "value" : "HTTP/1.1 200 OK"
        }, {
          "name" : "transfer-encoding",
          "value" : "chunked"
        }, {
          "name" : "Server",
          "value" : "nginx"
        }, {
          "name" : "Connection",
          "value" : "keep-alive"
        }, {
          "name" : "Date",
          "value" : "Fri, 17 Apr 2020 15:28:27 GMT"
        }, {
          "name" : "Content-Type",
          "value" : "application/json; charset=UTF-8"
        } ],
        "content" : {
          "size" : 1203,
          "mimeType" : "application/json; charset=UTF-8",
          "text" : "[\n  {\n    \"id\": \"9\",\n    \"active\": true\n  }\n]",
          "comment" : ""
        },
        "redirectURL" : "",
        "headersSize" : 131,
        "bodySize" : 1203,
        "comment" : ""
      },
      "cache" : { },
      "timings" : {
        "comment" : "",
        "blocked" : -1,
        "connect" : 715,
        "ssl" : -1,
        "send" : 0,
        "dns" : -1,
        "receive" : 0,
        "wait" : 0
      },
      "comment" : "",
      "_RequestInformation" : {
        "name" : "0",
        "testObjectId" : " environments.rs",
        "harId" : "87bc74eb-aa0f-4abf-a4a6-fd5fd639a893",
        "reportFolder" : null
      },
      "time" : 715
    } ],
    "comment" : ""
  }
}

**Comment by Huachao** [comment]
@kildarr  nice suggestion, I am considering the feature that supports importing requests of postman, swagger and HAR format.

**Comment by kildarr** [comment]
Actually what I'm asking for is to produce the HAR file for viewing or
export.

{
  "log" : {
    "version" : "1.2",
    "pages" : [ ],
    "entries" : [ {
      "startedDateTime" : "2020-04-20T15:00:50.755Z",
      "request" : {
        "method" : "GET",
        "url" : "
https://xxxxxxxxxxxxxxxxxxxxxxxxxx/v1/clients/8c9c26a9-3648-48ac-88b9-5545fce2261b/environments
",
        "httpVersion" : "",
        "cookies" : [ ],
        "headers" : [ {
          "name" : "Authorization",
          "value" : "Bearer 964da377-3d2c-429f-b158-abc9eb3c49d1"
        } ],
        "queryString" : [ ],
        "headersSize" : 0,
        "bodySize" : 0,
        "comment" : ""
      },
      "response" : {
        "status" : 200,
        "statusText" : "",
        "httpVersion" : "",
        "cookies" : [ ],
        "headers" : [ {
          "name" : null,
          "value" : "HTTP/1.1 200 OK"
        }, {
          "name" : "transfer-encoding",
          "value" : "chunked"
        }, {
          "name" : "Server",
          "value" : "nginx"
        }, {
          "name" : "Connection",
          "value" : "keep-alive"
        }, {
          "name" : "Date",
          "value" : "Mon, 20 Apr 2020 15:00:51 GMT"
        }, {
          "name" : "Content-Type",
          "value" : "application/json; charset=UTF-8"
        } ],
        "content" : {
          "size" : 1203,
          "mimeType" : "application/json; charset=UTF-8",
          "text" : "[\n  {\n    \"id\": \"9.........................":
true\n  }\n]",
          "comment" : ""
        },
        "redirectURL" : "",
        "headersSize" : 131,
        "bodySize" : 1203,
        "comment" : ""
      },
      "cache" : { },
      "timings" : {
        "comment" : "",
        "blocked" : -1,
        "ssl" : -1,
        "dns" : -1,
        "send" : 0,
        "wait" : 0,
        "connect" : 734,
        "receive" : 0
      },
      "comment" : "",
      "_RequestInformation" : {
        "name" : "2",
        "testObjectId" : "environments.rs",
        "harId" : "883431c4-98e1-4401-b93e-9f7321662629",
        "reportFolder" : null
      },
      "time" : 734
    } ],
    "comment" : ""
  }
}

**Comment by Huachao** [comment]
Another feature, sorry for the misunderstanding

**Comment by kildarr** [comment]
yes,  I would think we'd only want to generate the HAR file for the executed request; not everything in the file.  So maybe add a menu bar item on the response pane.
